webapp/src/main/webapp/templates/freemarker/edit/forms/addRoleToPersonTwoStage.ftl

```diff
@@ -195,12 +195,12 @@ Set this flag on the input acUriReceiver where you would like this behavior to o
             <#else>
                 <h4 class="label">${i18n().years_participating} </h4>
                 <#if htmlForElements?keys?seq_contains("startField")>
-                	    <label class="dateTime" for="startField">${i18n().start_capitalized}</label>
+                	    <label class="dateTime" for="startField">${i18n().start_year}</label>
                		    ${htmlForElements["startField"]} ${yearHint}
```

Comment:
This is producing (in adding head of or member of):
Start Year Year 

The word year is two times. The same is for end_year (after adding end_year in the property file)

Reply:
also, a place holder for Organization name has some duplicated text in adding role -  a member of.
Select an existing Select a type or create a new one

Reply:
For Creation of advisor relationship, in the placeholder Advisee and Subject Field are not internationalized